### ISSUE TITLE
The loss function does not appear to be converging.

### ISSUE THREAD

**Issue by sugar-Steamedfish** [Fourier7754/AsymFormer#14]
After training for 500 epochs, the loss function remains as shown in the figure. Does this mean it hasn't converged?
<img width="1138" height="995" alt="Image" src="https://github.com/user-attachments/assets/7622cc04-af20-456c-a21a-662f2e363fc5" />
